packages/hooks/src/useCurrencyExcahngeRate.ts

```diff
@@ -0,0 +1,10 @@
+import { useStore } from '@deriv/stores';
+
+const useCurrencyExcahngeRate = (currency: string) => {
+    const { exchange_rates } = useStore();
+    const rate = exchange_rates?.data?.rates?.[currency] || 1;
```

Comment:
also in case that the provided `currency` is not listed in rates object it returns undefined instead of `1` so please change in a way that if it couldn't find the value then it return the `1`